detox/src/devices/IosDriver.js

```diff
@@ -3,9 +3,9 @@ const fs = require('fs');
 const DeviceDriverBase = require('./DeviceDriverBase');
 const InvocationManager = require('../invoke').InvocationManager;
 const invoke = require('../invoke');
-const GREYConfigurationApi = require('./../ios/earlgreyapi/GREYConfiguration');
-const GREYConfigurationDetox = require('./../ios/earlgreyapi/GREYConfigurationDetox');
-const EarlyGrey = require('./../ios/earlgreyapi/EarlGrey');
+const GREYConfigurationApi = require('./../ios/earlgreyapi/GREYConfiguration'); // eslint-disable-line node/no-unpublished-require
```

Comment:
Please try to understand why this eslint doesn't like those lines. It makes no sense.

Reply:
I can remove it, should be red on master too, right 🤷‍♂️ The explanation with the npmignore makes kinda sense though